### DISCUSSION TITLE
Context menu

### DISCUSSION THREAD

**Discussion by Rootkit-** [WinMerge/winmerge#1152] [Q&A]
Windows 10 21h1
Things I've tried... 
Reinstalling and removing all winmerge related HKClasses roots.
Rebooting...

Results from current install of registry.
![image](https://user-images.githubusercontent.com/5389279/150676184-006a1507-6821-47bf-a95a-844a818251d6.png)

**Comment by Rootkit-** [comment]
![image](https://user-images.githubusercontent.com/5389279/150676252-5616048f-b753-4ccc-9948-787b9434532b.png)
![image](https://user-images.githubusercontent.com/5389279/150676259-69897df5-f576-4b2a-b7ac-8235f6617169.png)
![image](https://user-images.githubusercontent.com/5389279/150676265-0fe71640-dbc6-4edd-a59c-d68fc023b1c9.png)
what various context menus do look like... folder/file/2files

**Reply by sdottaka** [reply]
You reinstalled WinMerge, but WinMerge doesn't appear in Explorer's Context menu?

Is the Add to context menu checkbox checked in the Shell Integration category of WinMerge's Options dialog?

![image](https://user-images.githubusercontent.com/98126/150676682-ec645f10-34a2-421d-8b8e-67d6cd29531c.png)

**Comment by Rootkit-** [comment]
Yes all of those are done and toggled multiple times.

**Reply by sdottaka** [reply]
You seem to have installed the 32bit version of WinMerge, could you try **uninstalling** it and installing the 64bit version below?

https://github.com/WinMerge/winmerge/releases/download/v2.16.18/WinMerge-2.16.18-x64-Setup.exe
